main.go

```diff
@@ -242,8 +245,16 @@ func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *rea
 	// Block until the setup (certificate generation) finishes.
 	<-setupFinished
 
-	// initialize OPA
-	driver := local.New(local.Tracing(false), local.DisableBuiltins(disabledBuiltins.ToSlice()...))
+	var driver drivers.Driver
+	var providerCache *frameworksexternaldata.ProviderCache
+	if *externaldata.ExternalDataEnabled {
```

Comment:
nit: to avoid code duplication, you can start an `args` array, and add the `AddExternalDataProviderCache` flag to that array if it's enabled.

Then you can call:

```
driver = local.New(args...)
```

Reply:
thanks! added